packages/core/src/components/callout/_callout.scss

```diff
@@ -8,7 +8,7 @@ Callout
 
 Markup:
 <div class="#{$ns}-callout {{.modifier}}">
-  <h4 class="#{$ns}-heading">Callout Heading</h4>
+  <h5 class="#{$ns}-heading">Callout Heading</h5>
```

Comment:
In line with the [dialog update](https://github.com/palantir/blueprint/pull/5753), reducing the heading level to fit better in dense UIs.